R/tbl_split.R

```diff
@@ -40,27 +129,190 @@ tbl_split.gtsummary <- function(x, variables, ...) {
   # adding last variable
   variables <- variables |> union(dplyr::last(x$table_body$variable))
 
+  # footnotes and caption check
+  footnotes <- rlang::arg_match(footnotes)
+  caption <- rlang::arg_match(caption)
```

Comment:
We don't need to namespace rlang, because we import it.

Reply:
Right!!